map.h

```diff
@@ -2,94 +2,52 @@
 #include "splashkit.h"
 #include <fstream>
 #include <iostream>
+#include <memory>
+#include "block.h"
 using namespace std;
-// Define screen size and tilesize
-#define SCREEN_WIDTH  1600 // 50 tiles
-#define SCREEN_HEIGHT  896 // 28 tiles
-#define TILESIZE        16 // pixels
-#define MAP_WIDTH       40 // tiles
-#define MAP_HEIGHT      32 // tiles
 
 /**
  * @brief Map class
  * 
  */
-class Map
+class Level_Ojects_Map
```

Comment:
Should this be LevelObjectsMap to syntactically match other class names? (i.e FloorBlocks etc)